kolibri/core/errorreports/middleware.py

```diff
@@ -1,12 +1,41 @@
+import importlib.metadata
```

Comment:
This needs to be conditionalized based on Python version:
https://github.com/learningequality/kolibri/blob/542d558aabe4ca2c0dc190b857c79dcb744b4cc5/kolibri/plugins/utils/__init__.py#L15